data/mk_folders.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
I usually expect the data folder to contain nothing but the data.  So, I wouldn't have expected to look there if I were looking for code (and I might expect to be able to iterate over the directory in some circumstances).

In theory, you could move this file to another directory... but it'd be more common to put stuff like these 4 lines directly in the Makefile underneath the appropriate targets.